### ISSUE TITLE
Nuitka website down

### ISSUE THREAD

**Issue by feluxe** [Nuitka/Nuitka-website#9]
I just wanted to report, that nuitka.net seems to be down.

> The connection has timed out

**Comment by kayhayen** [comment]
I moved the website to another host and updated the DNS. Usually it takes 12-24h to make it switch over for everybody.

I finally found out why it was happening. Every since I upgraded to Debian Stretch, the kernel was complaining about wrong BIOS, and not being capable to control the power management, which apparently causes it to get hot, and this breaks the hardware to the point that it is now falling apart despite being idle.

I am going to sporadically reset it until the DNS must have switched over. Currently it is running, however system life times seem to have become very short.

I am leaving this open until the switch over is complete.
